### DISCUSSION TITLE
Make google image search crawl next/image

### DISCUSSION THREAD

**Discussion by jlabeit** [vercel/next.js#26286] [Help]
**Some background**
I've recently switched to the new `next/image` component for a larger project. Overall I was pretty happy with it - however then we've realized that all the images that are part of the site are not indexed anymore by google image search. As google image search is a source of traffic to our site I was wondering what the options are here to get google to index the images again.

- First, I through it had to do with lazy loading and that the images do not show up when google renders the page. However after investigating and running puppeteer like described [in this guide](https://developers.google.com/search/docs/guides/lazy-loading) the images seem to load properly.
- Then I read that "google only indexes images with proper URLs". I tried to dig into this a bit more but couldn't find-out for sure that this is the issue. 

**Ideas for solutions**
This is - depending on the device of course - how the image tags are loaded. As I understand the issue is that the `src` urls are "proper urls" for the google crawler and thus not indexed for image search.
```html
<img
  title="Title of the image"
  alt="some image"
  src="/_next/image?url=%2Fstatic%2Fimages%2Fsomeimage.png&amp;w=1920&amp;q=75"
  decoding="async"
/>
``` 
As I understand the issue I have two possible solution to get google to properly index the images:
1. I can remove image optimization (stop using `next/image`) and basically go back to plain old html with static unoptimized image urls. This would again have drawbacks for webvitals, loading times, ...
2. I can generate an image source map that contains the static unoptimized image urls. As far as I can tell this is not possible out of the box and would mean either manually maintaining the source map or writing some custom code to generate it from the project.

Does anyone else run into a similar issue? I just found [this thread](https://www.reddit.com/r/nextjs/comments/l5xmra/how_seo_friendly_is_the_nextimage_for_google/) on reddit but the answers are not to helpful.

**Comment by jasondibenedetto** [comment]
Just bumping this one – also looking for suggestions on how best to handle this at the moment.

**Reply by jlabeit** [reply]
Maybe a quick update from my side. I still have not figured out a solution for us. I've implemented a custom sitemap (option 2) a while back but google has not indexed the images so far. I am not sure if this has to do that the sitemap is referencing the original image urls that are not rendered in the html when the crawler is crawling the pages.

**Reply by jasondibenedetto** [reply]
Thanks for the update @jlabeit. We have opted to just use own image component for the time being. I will update this thread if we happen to stumble upon a more sophisticated solution.

**Comment by edward-meister** [comment]
any news on this issue?
